### PR TITLE
added geo downloads

### PR DESCRIPTION
**Pull request by aspcanada**
for your review @talos

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -9,6 +9,9 @@
                                               # will cause persistence issues!!
       "./observatory-extension:/observatory-extension"
     ],
+    "ports": [
+      "5432:5432"
+    ],
```
Comment (talos) [inline docker-compose.yml]: Did you need this line to get things to run, or is this an accident?

**Comment by talos** [comment]
This looks good, and ran successfully except:

```
Scheduled 23 tasks of which:
* 1 present dependencies were encountered:
    - 1 SubsectionTags()
* 13 ran successfully:
    - 3 DownloadGeography(geog_lvl=cd_,ct_,pr_)
    - 3 Geography(geog_lvl=cd_,ct_,pr_)
    - 3 GeographyColumns(geog_lvl=cd_,ct_,pr_)
    - 3 ImportGeography(geog_lvl=cd_) ...
    - 1 SectionTags()
* 4 failed:
    - 2 DownloadGeography(geog_lvl=cds_,cma_)
    - 2 GeographyColumns(geog_lvl=cds_,cma_)
* 5 were left pending, among these:
    * 5 had failed dependencies:
        - 1 AllGeographies()
        - 2 Geography(geog_lvl=cds_,cma_)
        - 2 ImportGeography(geog_lvl=cds_) and ImportGeography(geog_lvl=cma_)
```

Should `cds_` and `cma_` actually not be in the `GEOGRAPHIES` dict?  We don't have names for them in the names dict. Or is something else going wrong that's preventing the download from working?  This is the error I got for `cma`:

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/luigi/worker.py", line 177, in run
    new_deps = self._run_get_new_deps()
  File "/usr/local/lib/python2.7/dist-packages/luigi/worker.py", line 115, in _run_get_new_deps
    task_gen = self.task.run()
  File "/bigmetadata/tasks/util.py", line 631, in run
    for _, coltarget in self.output().iteritems():
  File "/bigmetadata/tasks/util.py", line 649, in output
    for col_key, col in self.columns().iteritems():
  File "/bigmetadata/tasks/ca/statcan.py", line 105, in columns
    name=GEOGRAPHY_NAMES[self.geog_lvl],
KeyError: 'cma_'
```

I got the same error for `cds_`.

File: docker-compose.yml (hunk 1 above)
Comment (aspcanada) [inline docker-compose.yml]: I exposed pqsl to localhost so I could use pg admin

**Comment by aspcanada** [comment]
I fixed this error in sg-6. You can close this PR as I already branched sg-6 from this one

File: docker-compose.yml (hunk 1 above)
Comment (talos) [inline docker-compose.yml]: OK -- that's fine, but I'd rather keep it closed down on our test deployment.  Could you remove this and keep the change locally only?

**Comment by talos** [comment]
Cool.  I'll close this.

**Pull request closed without merging**